src/core/evaluator.js

```diff
@@ -2034,11 +2034,23 @@ class PartialEvaluator {
       }
       var font = textState.font;
       if (!(font.loadedName in seenStyles)) {
+        let ascent = font.ascent;
+        let descent = font.descent;
+        if (font.bbox) {
+          ascent = ascent || font.bbox[3];
+          descent = descent || -Math.abs(font.bbox[1]);
+        }
+        if (ascent && !isNaN(descent)) {
```

Comment:
Don't you need to *also* guard against the divisor becoming zero here, since an ascent value of `Infinity` seems less than useful?
```suggestion
        if (ascent && !isNaN(descent) && ascent !== descent) {
```

Reply:
Isn't the descent supposed to be a negative or zero number ? And the ascent a positive or zero one ?

Reply:
Generally speaking that's obviously how these things *should* behave, but given all the stupid things that PDF generators tend to do I wanted to at least ask about this :-)